server/views/index.ts

```diff
@@ -25,31 +31,54 @@ export function renderIndex() {
         window.process.env.API_URL = '${process.env.API_URL}';
         window.process.env.NODE_ENV = '${process.env.NODE_ENV}';
         window.process.env.S3_BUCKET = '${process.env.S3_BUCKET}';
-        window.process.env.S3_BUCKET_URL_BASE = ${process.env.S3_BUCKET_URL_BASE ? `'${process.env.S3_BUCKET_URL_BASE}'` : undefined};
+        window.process.env.S3_BUCKET_URL_BASE = ${
+          process.env.S3_BUCKET_URL_BASE
+            ? `'${process.env.S3_BUCKET_URL_BASE}'`
+            : undefined
+        };
         window.process.env.AWS_REGION = '${process.env.AWS_REGION}';
-        window.process.env.FORCE_TO_HTTPS = ${process.env.FORCE_TO_HTTPS === 'false' ? false : undefined};
+        window.process.env.FORCE_TO_HTTPS = ${
+          process.env.FORCE_TO_HTTPS === 'false' ? false : undefined
+        };
         window.process.env.CLIENT = true;
-        window.process.env.LOGIN_ENABLED = ${process.env.LOGIN_ENABLED === 'false' ? false : true};
-        window.process.env.EXAMPLES_ENABLED = ${process.env.EXAMPLES_ENABLED === 'false' ? false : true};
-        window.process.env.UI_ACCESS_TOKEN_ENABLED = ${process.env.UI_ACCESS_TOKEN_ENABLED === 'false' ? false : true};
-        window.process.env.UI_COLLECTIONS_ENABLED = ${process.env.UI_COLLECTIONS_ENABLED === 'false' ? false : true};
-        window.process.env.UPLOAD_LIMIT = ${process.env.UPLOAD_LIMIT ? `${process.env.UPLOAD_LIMIT}` : undefined};
-        window.process.env.TRANSLATIONS_ENABLED = ${process.env.TRANSLATIONS_ENABLED === 'true' ? true : false}; 
+        window.process.env.LOGIN_ENABLED = ${
+          process.env.LOGIN_ENABLED !== 'false'
```

Comment:
no big deal either way but would you want to use the helper utils you wrote for this case? https://github.com/processing/p5.js-web-editor/blob/2336986630d64a49378869e5cc7c0aa9a2c8fb72/client/utils/parseStringToType.ts#L33

the only thing is you'd have to move the utility file into common/